ONIT.VismaNet/Models/ShipmentDetailLine.cs

```diff
@@ -2,18 +2,19 @@
 using ONIT.VismaNetApi.Lib;
 using ONIT.VismaNetApi.Models.CustomDto;
 using ONIT.VismaNetApi.Models.Enums;
+using System.Collections.Generic;
 
 namespace ONIT.VismaNetApi.Models
 {
     public class ShipmentDetailLine : DtoProviderBase
     {
         public ShipmentDetailLine()
         {
-            DtoFields.Add("operation", new NotDto<ApiOperation>(ApiOperation.Insert));
+            DtoFields.Add("operation", new NotDto<ApiOperation>(ApiOperation.Update));
```

Comment:
These should be Insert. Use the PrepareForUpdate on ShipmentDetail to set all retrieved lines to Update.